lib/ovirt/service.rb

```diff
@@ -199,7 +199,7 @@ def resource_verb(path, verb, *args)
       logger.debug "#{log_header}: With args: <#{args.inspect}>"
       resource.send(verb, *args) do |response, request, result, &block|
         case response.code
-        when 200
+        when 200..206
```

Comment:
200 returns shouldn't require any special processing.  We receive the same response, but now it has the correct response code.